foundations/html_css/html-foundations/project-recipes.md

```diff
@@ -48,11 +48,11 @@ Okay, that's enough Git for the moment -- time to actually build stuff!
 1. Create a new directory within the `odin-recipes` directory and name it `recipes`.
 2. Create a new HTML file within the  `recipes` directory and name it after the recipe it will contain. For example `lasagna.html`. You can use the name of your favorite dish or, if you need some inspiration, you can [find a recipe to use here](https://www.allrecipes.com/).
 3. For now, just include an `h1` heading with the recipe's name as its content.
-4. Back in the `index.html` file, add a link to the recipe page you just created. The text of the link should again be the recipe name.
+4. Back in the `index.html` file, add a link to the recipe page you just created. Example: Under the <h1>Odin Recipes</h1> heading, write out the link like so: <a href="recipename.html">Recipe Title</a>. The text of the link should again be the recipe name.
```

Comment:
```suggestion
4. Back in the `index.html` file, add a link to the recipe page you just created. Example: Under the `<h1>Odin Recipes</h1>` heading, write out the link like so: `<a href="recipename.html">Recipe Title</a>`. The text of the link should again be the recipe name.
```
Without backticks, your code examples will get parsed into actual page elements.
Please do next time try out your additions with the Markdown Preview tool. That's why it's listed as a requirement.

Reply:
Hey Zach thank you for your comment! I'll be sure to add those code blocks in the future!